low-level/framework/platforms/server/network_interface.py

```diff
@@ -0,0 +1,81 @@
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com
+
+import os
+
+from framework.utils.conf_utils import (
+    Conf, SSPL_CONF, SYSFS_PATH, SYSTEM_INFORMATION)
+from framework.platforms.server.error import InterfaceError
+
+
+class NetworkInterface:
```

Comment:
We can rename this to `NetworkPorts` as well if that seems more appropriate.

Reply:
'class Network' covers wider scope, be it interface or cable connection